modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -74,10 +74,10 @@ def send(at: nil)
       return unless SUPPORTED_FORMS.include?(form_number)
 
       data = form_specific_data || empty_form_specific_data
-
       return if data[:email].blank? || data[:personalization]['first_name'].blank?
 
       template_id = TEMPLATE_IDS[form_number][notification_type]
+      return unless template_id
```

Comment:
If we return early here, do we want to raise or log an exception?

Reply:
No, there are cases where we don't have a template_id for a certain notification type and in such cases it means we don't want to send an email. This isn't a failure mode, this is by design.